synapse/federation/sender/transaction_manager.py

```diff
@@ -54,7 +54,10 @@ def __init__(self, hs: "synapse.server.HomeServer"):
 
     @measure_func("_send_new_transaction")
     async def send_new_transaction(
-        self, destination: str, pending_pdus: List[EventBase], pending_edus: List[Edu]
+        self,
+        destination: str,
+        pending_pdus: List[Tuple[EventBase, int]],
```

Comment:
I'm guessing this was also found from adding `measure_func` types?

Reply:
Yes, sorry